pkg/config/config.go

```diff
@@ -169,12 +169,13 @@ type (
 	}
 
 	studio struct {
-		Enabled      bool   `toml:"enabled"`
-		Image        string `toml:"-"`
-		Port         uint16 `toml:"port"`
-		ApiUrl       string `toml:"api_url"`
-		OpenaiApiKey Secret `toml:"openai_api_key"`
-		PgmetaImage  string `toml:"-"`
+		Enabled         bool   `toml:"enabled"`
+		Image           string `toml:"-"`
+		Port            uint16 `toml:"port"`
+		ApiUrl          string `toml:"api_url"`
+		OpenaiApiKey    Secret `toml:"openai_api_key"`
+		PgmetaImage     string `toml:"-"`
+		PgmetaCryptoKey string `toml:"-"`
```

Comment:
I figure `PgmetaCryptoKey` belongs here since `PgmetaImage` is already there, but was wondering if it's better to split the pgmeta stuff into its own struct.